### PR TITLE
Validate eligibility value for funded places

### PR DESCRIPTION
**Pull request by padv2010**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CPDLP-3078

### Description

Update the admin tool in the NPQ tab so it allows updating the eligible for funding value, where we should return an error if the funded place is true and we attempt to set the eligible for funding to false.

### CODE REVIEW

**Review by cwrw — APPROVED** [review]

File: spec/models/npq_application_spec.rb [hunk 1]
```diff
@@ -353,10 +353,36 @@ def subject.transient_previously_funded
...
+        expect(npq_application.valid?).to eq(true)
+      end
+
+      it "is valid if `eligible_for_funding` is false and `funded_ place` is false" do
+        npq_application.eligible_for_funding = false
+        npq_application.funded_place = false
 
+        expect(npq_application.valid?(:admin)).to eq(true)
+      end
+      it "is valid if `eligible_for_funding` is false and `funded_ place` is nil" do
```
Comment (cwrw) [inline spec/models/npq_application_spec.rb:377]: could we add a test for true/true, and true/nil?

Comment (padv2010) [reply spec/models/npq_application_spec.rb:378]: Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/4863/commits/f5745404f8659707312308a12abfe78ce453e83a

**Review by cwrw — APPROVED** [review]
